apps/stripe/src/app/api/saleor/payment-gateway-initialize-session/use-case-response.ts

```diff
@@ -0,0 +1,36 @@
+import { buildSyncWebhookResponsePayload } from "@saleor/app-sdk/handlers/shared";
+import { z } from "zod";
+
+import { SuccessWebhookResponse } from "@/modules/saleor/saleor-webhook-responses";
+import { StripePublishableKey } from "@/modules/stripe/stripe-publishable-key";
+
+class Success extends SuccessWebhookResponse {
+  pk: StripePublishableKey;
```

Comment:
if you pass it to constructor, better make it readonly or private. Mutation of class member can lead to hard to find bugs and I dont think its our intention